etcdserver/api/snap/snapshotter.go

```diff
@@ -282,12 +282,13 @@ func (s *Snapshotter) ReleaseSnapDBs(snap raftpb.Snapshot) error {
 			hexIndex := strings.TrimSuffix(filepath.Base(filename), ".snap.db")
 			index, err := strconv.ParseUint(hexIndex, 16, 64)
 			if err != nil {
-				return fmt.Errorf("failed to parse index from .snap.db filename %q: %v", filename, err)
+				s.lg.Error("failed to parse index from filename", zap.String("path", filename), zap.String("error", err.Error()))
+				continue
 			}
 			if index < snap.Metadata.Index {
 				s.lg.Info("found orphaned .snap.db file; deleting", zap.String("path", filename))
 				if rmErr := os.Remove(filepath.Join(s.dir, filename)); rmErr != nil && !os.IsNotExist(rmErr) {
-					return fmt.Errorf("failed to remove orphaned defragmentation file %s: %v", filename, rmErr)
+					s.lg.Error("failed to remove orphaned defragmentation file", zap.String("path", filename), zap.String("error", rmErr.Error()))
```

Comment:
Would you change this to say "failed to removed orphaned .snap.db file" ? This is my fault and I just noticed it now :(

Reply:
Addressed.